README.md

```diff
@@ -9,7 +9,7 @@ A CodePen demo is available [here](http://codepen.io/davidchin/full/GpNvqw/).
 
 ## Installation
 
-1. Install `react-input-range` using npm. `npm install react-input-range`
+1. Install `react-input-range` using yarn (or npm). `yarn install react-input-range`
```

Comment:
I think the sentence should read

> Install `react-input-range` using npm (or yarn). `npm install react-input-range`

It is better to use `npm` as the example because it is the defacto node package manager. When people use this library, they can choose to use `yarn` if they want to. But it's not required.